src/encoded/audit/experiment.py

```diff
@@ -3609,20 +3618,27 @@ def audit_experiment_spikeins(value, system, excluded_types):
         if lib is None:
             continue
 
-        size_range = lib.get('size_range')
-        if size_range != '>200':
+        if 'size_range' in lib:
+            size_range = lib.get('size_range')
+            if size_range != '>200':
+                continue
+        elif 'average_fragment_size' in lib:
+            size_range = lib.get('average_fragment_size')
+            if size_range <= 200:
+                continue
+        else:
             continue
```

Comment:
this is exact repetition of a code above, I think you can extract it into a function.

Reply:
Good idea, I had both sections call a small function instead